components/eamxx/src/physics/shoc/eamxx_shoc_process_interface.cpp

```diff
@@ -111,7 +111,7 @@ void SHOCMacrophysics::set_grids(const std::shared_ptr<const GridsManager> grids
   } // Extra SHOC output diagnostics
 
   // Tracer group
-  add_group<Updated>("tracers", grid_name, ps, true);
+  add_group<Updated>("turbulence_advected_tracers", grid_name, ps, true);
```

Comment:
I know we just removed it, but in the spirit of self-documenting code: do you think we should bring back the enum for requesting bundling? I'm wondering if it's worth it (folks would read the line above with something like `BundlingNeeded` or `MonolithicField` or whatever instead of `true`), or if it's ok to assume folks would just go and see what the signature of `add_group` is...

Reply:
See comment above.